openshift-install-powervs

```diff
@@ -60,9 +61,9 @@ EOF
 
 RELEASE_VER=${RELEASE_VER:-"4.6"}
 ARTIFACTS_REPO=${ARTIFACTS_REPO:-"https://github.com/ocp-power-automation/ocp4-upi-powervs"}
-ARTIFACTS_VERSION=${ARTIFACTS_VERSION:-"release-$RELEASE_VER"}
+#ARTIFACTS_VERSION=${ARTIFACTS_VERSION:-"release-$RELEASE_VER"}
```

Comment:
Why this change? This is not needed. Master is used only for dev preview releases.

Reply:
Updated this change.